src/Schema/Blueprint.php

```diff
@@ -296,6 +315,57 @@ public function dropRowDeletionPolicy(): Fluent
         return $this->addCommand('dropRowDeletionPolicy');
     }
 
+    /**
+     * @param string $name
+     * @return SequenceDefinition
```

Comment:
Perhaps `Fluent<string, mixed>` would be more appropriate?

Reply:
SequenceDefinition is correct here, it was just not passed correctly, fixed.